userguide/content/en/docs/Getting started/_index.md

```diff
@@ -255,6 +262,30 @@ gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
 To use your own Custom Search Engine, replace the value in the `gcs_engine_id` with the ID of your own search engine.
 
+## Known issues
+
+### MacOS
+
+#### `too many open files` - File Descriptors for "hugo server"
```

Comment:
Should we maybe change this to the name of the top-level error that users see when running `hugo server` - what actually shows up is `pipe failed`?

So something like:

#### `fatal error: pipe failed` error with `hugo server`

Reply:
I believe this depends on your version of postcss. pre v8, you will see the `Error: too many open files` message, which is why i commented that if it worked, we would add it. [Example](https://github.com/gohugoio/hugo/issues/7773)

That said, it sounds like Knative (we are floating around 2800 files per doc version) is the only project that size who have seen the later version postcss errors.

Should we include both? Or just address the latest version?

Reply:
I think all the users I've seen have had the `fatal error: pipe failed` error but yeah, if that's the error you got and your workaround fixes both then we should do both.

So how about for the title:

`too many open files` or `fatal error: pipe failed` errors with `hugo server`

?